src/series/groupedSeries.js

```diff
@@ -37,55 +37,69 @@ export default function() {
 
             g.select('g')
                 .datum(values)
-                .each(function(series, index) {
+                .each(function(_series, index) {
```

Comment:
A common naming convention for function arguments that are not used is simply underscore, `_`